flexibleengine/resource_flexibleengine_elb_loadbalancer.go

```diff
@@ -229,7 +229,7 @@ func resourceELoadBalancerDelete(d *schema.ResourceData, meta interface{}) error
 
 	id := d.Id()
 	log.Printf("[DEBUG] Deleting loadbalancer %s", d.Id())
-	job, err := loadbalancer_elbs.Delete(client, id).ExtractJobResponse()
+	job, err := loadbalancer_elbs.Delete(client, id, false).ExtractJobResponse()
```

Comment:
I guess in the newer API you can retain the elastic IPs, this should be approved by a maintainer

Reply:
+1